aten/src/ATen/native/nested/NestedTensorFactories.cpp

```diff
@@ -106,9 +106,26 @@ Tensor _to_copy_nested(
   Tensor r;
   r = at::empty_like(self, dtype, layout, device, pin_out, memory_format);
   get_nested_tensor_impl(r)->get_buffer().copy_(
-      get_nested_tensor_impl(self)->get_buffer());
+      get_nested_tensor_impl(self)->get_buffer(), non_blocking);
   return r;
 }
 
+Tensor& copy_nested_(Tensor& self, const Tensor& src, bool non_blocking) {
+  get_nested_tensor_impl(self)->get_buffer().copy_(get_nested_tensor_impl(src)->get_buffer(), non_blocking);
+  return self;
+}
+
+Tensor& fill_nested_(Tensor& self, const Scalar& value) {
+  const auto& self_buf = get_nested_tensor_impl(self)->get_buffer();
+  self_buf.fill_(value);
+  return self;
+}
+
+Tensor& fill_nested_(Tensor& self, const Tensor& value) {
+  const auto& self_buf = get_nested_tensor_impl(self)->get_buffer();
+  self_buf.fill_(value);
```

Comment:
This won't work if value is anything but a scalar. In the test you only all it with scalar. Do you need to support this overload for your use case?

Reply:
No. I can add the test that fails with a non-0-dimensional tensor too. I think the reason this exists is so that you can do something like t.fill_(other_t[0]), which returns a 0-dimensional tensor instead of a scalar.